demo/demo-spring-boot-transport/demo-spring-boot-pojo-client/pom.xml

```diff
@@ -42,6 +46,11 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-netflix-core</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
client depend on server
then service's microservice.yaml in classpath too, which one will override another one, it's random, except you point the order.
even you point the order, it's bad practice.

all server schema deployed to client too. do you want this?

Reply:
I have fixed this problem by removing the dependency in client side, and organized import of RestServletInitializer.java. Please review this commit.